deploy/crds/integreatly_v1alpha1_redis_cr.yaml

```diff
@@ -11,4 +11,4 @@ spec:
   # i want a redis storage of a development-level tier
   tier: development
   # i want a redis storage for the type managed
-  type: managed
```

Comment:
^^ :eyes:

Reply:
Can we also make the `secretRef` names the same across all CRs?